<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>relop</class>
 <widget class="QDialog" name="relop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relación de operaciones con terceras personas según diario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton1">
          <property name="text">
           <string>Ad&amp;quisiciones de bienes y servicios</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton2">
          <property name="text">
           <string>&amp;Ventas de bienes y servicios</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>&lt;b&gt;A PARTIR DE IMPORTE:&lt;/b&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="importelineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>&lt;b&gt;EJERCICIO:&lt;/b&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ejerciciocomboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_3">
       <property name="text">
        <string>&lt;b&gt;Códigos (subgrupos, cuentas):&lt;/b&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codigoslineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="deduclabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>% Deducción por defecto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deduclineEdit">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="naturalezacomboBox">
     <item>
      <property name="text">
       <string>1. Entidad Beneficiaria incentivos Título III Ley 49/2002 de r.f. de entidades sin fines lucrativos e incentivos mecenazgo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2. Fundación o asociación declarada de utilidad pública art. 68.3.b) Ley IRPF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3. Titular o administrador de un patrimonio protegido Ley 41/2003 de protección patrimonial de personas con discapacidad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4. Partidos Políticos, Federaciones, Coaliciones o Agrupaciones de Electores</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabla">
     <property name="columnCount">
      <number>12</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cargapushButton">
       <property name="text">
        <string>&amp;Cargar datos</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/rayo.png</normaloff>:/iconos/graf/rayo.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrarpushButton">
       <property name="text">
        <string>&amp;Borrar línea</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton">
       <property name="text">
        <string>C&amp;opiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="datospushButton">
       <property name="text">
        <string>&amp;Datos cuenta</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/text-x-vcard.png</normaloff>:/iconos/graf/text-x-vcard.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m182pushButton">
       <property name="text">
        <string>&amp;Modelo 182</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/aeat.png</normaloff>:/iconos/graf/aeat.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimepushButton">
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="latexpushButton">
       <property name="text">
        <string>&amp;Editar Latex</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/tex.png</normaloff>:/iconos/graf/tex.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButton1</tabstop>
  <tabstop>radioButton2</tabstop>
  <tabstop>importelineEdit</tabstop>
  <tabstop>ejerciciocomboBox</tabstop>
  <tabstop>codigoslineEdit</tabstop>
  <tabstop>deduclineEdit</tabstop>
  <tabstop>naturalezacomboBox</tabstop>
  <tabstop>tabla</tabstop>
  <tabstop>cargapushButton</tabstop>
  <tabstop>borrarpushButton</tabstop>
  <tabstop>copiarpushButton</tabstop>
  <tabstop>datospushButton</tabstop>
  <tabstop>m182pushButton</tabstop>
  <tabstop>imprimepushButton</tabstop>
  <tabstop>latexpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>relop</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>993</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
